docs/root/start/sandboxes/locality_load_balancing.rst

```diff
@@ -0,0 +1,141 @@
+.. _install_sandboxes_locality_load_balancing:
+
+Locality Weighted Load Balancing
+================================
+
+.. sidebar:: Requirements
+
+   .. include:: _include/docker-env-setup-link.rst
+
+   :ref:`curl <start_sandboxes_setup_curl>`
+        Used to make ``HTTP`` requests.
+
+This example creates a setup for demonstrating the :ref:`locality weighted load balancing <arch_overview_load_balancing_locality_weighted_lb>` feature in Envoy proxy. The demo simulates a scenario that a backend service resides in two local zones and one remote zone.
+
+The components used in this demo are as follows:
+
+- A client container: runs Envoy proxy
+- Backend container in the same locality as the client, with priority set to 0, namely ``local-1``.
+- Backend container in the same locality as the client, with priority set to 1, namely ``local-2``.
+- Backend container in the the remote locality, with priority set to 1, namely ``remote-1``.
+- Backend container in the the remote locality, with priority set to 2, namely ``remote-2``.
+
+The client Envoy proxy configures the 4 backend containers in the same Envoy cluster, so that Envoy handles load balancing to those backend servers.
+
+Step 1: Start all of our containers
+***********************************
+
+In terminal, move to the ``examples/locality_load_balancing`` directory.
+
+To build this sandbox example and start the example services, run the following commands:
+
+.. code:: shell
+
+    # Start demo
+    $ docker-compose up --build -d
+
+The locality configuration is set in the client container via static Envoy configuration file. Please refer to the :download:`clusters configuration <_include/locality-load-balancing/envoy-client.yaml>` file.
+
+Step 2: Scenario with one replica in the highest priority locality
+******************************************************************
+
+In this scenario, each locality has 1 healthy replica running. In this case, all the requests should be sent to the locality with the highest priority, which is ``local-1``.
+
+.. code:: shell
+
+    # all requests to local-1
+    $ docker-compose exec -T client-envoy python3 client.py http://localhost:3000/ 100
+    Hello from backend-local-1!: 100, 100.0%
+    Failed: 0
+
+If locality ``local-1`` becomes unhealthy, the requests should be load balanced among the subsequent priority localities, which are ``local-2`` and ``remote-1``. They both have priority 1. We then send 100 requests to the backend cluster, and check the responders.
```

Comment:
might be helpful to have a simple explanation of what becoming unhealthy is - ie returning 503s

Reply:
Will update here. Becoming unhealthy here means failed the Envoy health check.